Tests/LibAudio/CMakeLists.txt

```diff
@@ -1,10 +1,13 @@
 set(TEST_SOURCES
+    TestWav.cpp
     TestFLACSpec.cpp
-    TestPlaybackStream.cpp
+    TestPlaybackStream.cpp    
 )
 
 foreach(source IN LISTS TEST_SOURCES)
     serenity_test("${source}" LibAudio LIBS LibAudio)
 endforeach()
 
 install(DIRECTORY ${FLAC_SPEC_TEST_PATH} DESTINATION usr/Tests/LibAudio/FLAC)
+
+install(DIRECTORY wav-test-files DESTINATION usr/Tests/LibAudio)
```

Comment:
To keep with the pattern I have established with FLAC, let's rename this folder to simply "WAV".